index.html

```diff
@@ -4,27 +4,29 @@
     <link rel="stylesheet" type="text/css" href="css/normalize.css">
     <link rel="stylesheet" type="text/css" href="css/styles.css">
     <link rel="stylesheet" type="text/css" href="css/splash.css">
+    <link rel="stylesheet" type="text/css" href="css/track.css">
     <link href="https://fonts.googleapis.com/css?family=Press+Start+2P" rel="stylesheet">
 </head>
 <body>
-    <main class="container">
-        <div class="splash">
-            <img src="assets/gameTitle.png" alt="Mayrio Kart">
+    <main id="container" class="container">
```

Comment:
having a class and an id with the same name on the same element is not verbose

Reply:
Does it need a both? could it just be referenced as ```<main>``` in the css as its the only one?

Reply:
we omitted the class element, retained the id and changed the css styles files accordingly